lib/middleware/common.js

```diff
@@ -1,41 +1,44 @@
 /**
  * This module contains some common helpers shared between middlewares
  */
+'use strict'
 
-var mime = require('mime')
-var _ = require('lodash')
-var parseRange = require('range-parser')
-var Buffer = require('safe-buffer').Buffer
+const mime = require('mime')
+const _ = require('lodash')
+const parseRange = require('range-parser')
+const Buffer = require('safe-buffer').Buffer
 
-var log = require('../logger').create('web-server')
+const log = require('../logger').create('web-server')
 
-var PromiseContainer = function () {
-  var promise
+class PromiseContainer {
+  constructor () {
+    this.promise = null
+  }
 
-  this.then = function (success, error) {
+  then (success, error) {
     error = error || _.noop
-    return promise.then(success).catch(error)
+    return this.promise.then(success).catch(error)
   }
 
-  this.set = function (newPromise) {
-    promise = newPromise
+  set (newPromise) {
+    this.promise = newPromise
   }
 }
 
-var serve404 = function (response, path) {
+const serve404 = function (response, path) {
   log.warn('404: ' + path)
   response.writeHead(404)
   return response.end('NOT FOUND')
 }
 
-var createServeFile = function (fs, directory, config) {
-  var cache = Object.create(null)
+const createServeFile = function (fs, directory, config) {
```

Comment:
What do you think about `function createServeFile (fs, directory, config)` instead of `const` ?

Reply:
done (also others)